packages/shipjs/src/flow/setup.js

```diff
@@ -19,15 +19,14 @@ async function setup({ help = false, dir = '.', dryRun = false }) {
   const {
     baseBranch,
     releaseBranch,
-    configureCircleCI,
-    scheduleCircleCI,
-    cronExpr,
     useMonorepo,
     mainVersionFile,
     packagesToBump,
     packagesToPublish,
     isScoped,
     isPublic,
+    CIIndex,
+    CIConfig,
   } = await askQuestions({ dir });
```

Comment:
What if `askQuestions` return `ciIntegration` and `ciConfig` instead of `CIIndex` and `CIConfig`?
What do you think?

(And, FYI, the camelCase of `CIConfig` is `ciConfig` 😃 )